anatomy/tracks/config-json.md

```diff
@@ -175,6 +169,46 @@ Tags are specified in the top-level `tags` field which is defined as an array of
 
 - `typing/static`: the language uses static typing
 - `typing/dynamic`: the language uses dynamic typing
+- `typing/strong`: the language uses strong typing
+- `typing/weak`: the language uses weak typing
+
+## Execution mode
+
+- `execution_mode/compiled`: code is compiled first before being executed
+- `execution_mode/interpreted`: code is interpreted directly
+
+## Platform
+
+- `platform/windows`: runs on Windows
+- `platform/mac`: runs on Mac
+- `platform/linux`: runs on Linux
+- `platform/ios`: runs on iOS
+- `platform/android`: runs on Android
+- `platform/web`: runs on Web
```

Comment:
What does `runs on Web` mean? Runs in a browser?

Reply:
It does. Would you prefer `platform/browser`?

Reply:
I think so

Reply:
Platform web runs on browser is fine I think